ui-ngx/src/app/modules/home/components/widget/dynamic-widget.component.ts

```diff
@@ -80,6 +81,7 @@ export class DynamicWidgetComponent extends PageComponent implements IDynamicWid
     this.ctx.dialogs = $injector.get(DialogService);
     this.ctx.customDialog = $injector.get(CustomDialogService);
     this.ctx.resourceService = $injector.get(ResourceService);
+    this.ctx.telemetryWsService = $injector.get(TelemetryWebsocketService);
```

Comment:
What is the need to add the context if it has already been added to the ServicesMap?